demos/src/data-inverse.json

```diff
@@ -10,6 +10,8 @@
         "withLinkedin": true,
         "withWhatsapp": true,
         "withPinterest": true,
+        "withEnterpriseSharing": false,
+        "withShare": true,
```

Comment:
`withShare` has its own demo, which explains that share behaviour is customisable and not build into o-share, I don't think we want to include it in other demos?

Reply:
This should be fixed in [`f596770` (#216)](https://github.com/Financial-Times/o-share/pull/216/commits/f596770f8bcdb9caea1fded4ba7d720cce54abbe)